app/models/google_classroom_course.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class GoogleClassroomCourse
+  attr_accessor :name, :students
```

Comment:
can we change this to `attr_reader` as we are not mutating their values on this object anywhere else